bokehjs/src/less/continuum.less

```diff
@@ -16,7 +16,7 @@
 .tableheader {
   background-color: #C0C0C0;
 }
-#notebook .bk-plot-wrapper table {
```

Comment:
So why this was bound to notebook structure before? Removing this implies that the following styles will be applied always. Is this intended?